migtests/scripts/functions.sh

```diff
@@ -230,17 +230,38 @@ run_sqlplus() {
 }
 
 export_schema() {
-	args="--export-dir ${EXPORT_DIR}
-		--source-db-type ${SOURCE_DB_TYPE}
-		--source-db-user ${SOURCE_DB_USER}
-		--source-db-password ${SOURCE_DB_PASSWORD}
+    # Default values
+    export_dir="${EXPORT_DIR}"
+    source_db_schema="${SOURCE_DB_SCHEMA}"
```

Comment:
add a comment before the function, explain how are the args passed to it, i.e. [args_name1 arg_val1...]